constants.cs

```diff
@@ -77,9 +77,14 @@ public static class Constants {
     // Error messages
     public const string ERROR_FILE_FORMAT = "ERROR: File format does not conform to format: <filename>.mp; ";
     public const string ERROR_DISPATCHER_DIGIT = "ERROR: Dispatcher done goofed. Passed {0} to the digit FSA, but {0} is not a digit.";
+    public const string ERROR_DISPATCHER_LETTERS = "ERROR: Dispatcher failed miserably. Passed {0} to the letter FSA, but {0} is not a valid identifier.";
```

Comment:
This goes over the 100 character limit, but we can deal with that in the refactor ticket.